pkg/controllers/provisioningv2/cluster/remove.go

```diff
@@ -142,5 +154,5 @@ func (h *handler) doClusterRemove(cluster *v1.Cluster) (string, error) {
 		return fmt.Sprintf("waiting for cluster-api cluster [%s] to delete", cluster.Name), nil
 	}
 
-	return "", nil
+	return "", h.kubeconfigManager.DeleteUser(cluster.Namespace, cluster.Name)
```

Comment:
wait... the user name is not the same as cluster.Name. You have to calculate through hash function

Reply:
oh nvm, I see it is a wrapper function

Reply:
It uses the same pattern as "CreateUser"